.github/workflows/release.yml

```
@@ -183,18 +233,26 @@ jobs:
-        uses: actions/upload-artifact@v4
-        env:
-          GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+      - name: Upload Artifact
+        uses: actions/upload-artifact@v4  # Fixed this block
         with:
           name: windows-build-debug
           path: apps/flipper/build/windows/x64/runner/Release/flipper_rw.msix
-          upload_release_asset: false
+
+      - name: Create Release
+        uses: softprops/action-gh-release@v1  # Or equivalent release action
```

actionlint (1.7.4)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)